library/ssl_tls13_server.c

```diff
@@ -708,7 +708,7 @@ int mbedtls_ssl_parse_client_psk_identity_ext(
                     if( diff > MBEDTLS_SSL_TICKET_AGE_TOLERANCE )
                     {
                         MBEDTLS_SSL_DEBUG_MSG( 3,
-                            ( "Ticket age outside tolerance window ( diff=%lld )",
+                            ( "Ticket age outside tolerance window ( diff=%ld )",
```

Comment:
Using [PRId64](https://stackoverflow.com/questions/9225567/how-to-print-a-int64-t-type-in-c) may be more portable.
```
#include <inttypes.h>

"Ticket age outside tolerance window ( diff=%" PRId64  ")",
```